### PR TITLE
2.03 description in doc link 357

### PR DESCRIPTION
**Pull request by amy-silcock**
Adding Extra Documentation as per: https://github.com/IATI/IATI-Schemas/issues/357

### CODE REVIEW

**Review by PetyaKangalova — APPROVED** [review]

**Review by amy-silcock — COMMENTED** [review]
@PetyaKangalova Looks good

**Comment by dalepotter** [comment]
Changed base branch in order to force a refresh to the commit and file diff. This PR previously included changes already in the base branch.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ en/_build
 */html
 
 
+en/activity-standard/activity-standard-example-annotated.xml
```
Comment (dalepotter) [inline .gitignore]: I'm not quite sure why this file has been added to `.gitignore`...?

**Review by dalepotter — CHANGES_REQUESTED** [review]

File: en/organisation-standard/iati-organisations/iati-organisation/document-link.rst [hunk 2]
```diff
@@ -28,3 +28,6 @@ Changelog
 ^^^^
 Removed ``@language`` attribute from, and introduced an new ``language`` child element to, the ``document-link`` element.
 
+2.03
```
Comment (dalepotter) [inline en/organisation-standard/iati-organisations/iati-organisation/document-link.rst]: I think the convention is to list changes reverse-chronologically - so here the `2.03` change would be listed before the `1.02` change.

**Review by dalepotter — CHANGES_REQUESTED** [review]

**Review by dalepotter — APPROVED** [review]

**Pull request merged**